src/actions/TokenTermsActions.tsx

```diff
@@ -4,19 +4,12 @@ import { sprintf } from 'sprintf-js'
 
 import { ConfirmContinueModal } from '../components/modals/ConfirmContinueModal'
 import { Airship } from '../components/services/AirshipInstance'
-import { TOKEN_TERMS_AGREEMENT } from '../constants/constantSettings'
 import { lstrings } from '../locales/strings'
 import { config } from '../theme/appConfig'
 
 export const approveTokenTerms = async (disklet: Disklet, currencyCode: string) => {
   const title = sprintf(lstrings.token_agreement_modal_title, currencyCode)
   const body = sprintf(lstrings.token_agreement_modal_message, currencyCode, config.appName)
-  const filePath = `${currencyCode}-${TOKEN_TERMS_AGREEMENT}`
 
-  try {
-    await disklet.getText(filePath)
-  } catch (error: any) {
-    await Airship.show<boolean>(bridge => <ConfirmContinueModal bridge={bridge} title={title} body={body} />)
-    await disklet.setText(filePath, '')
-  }
+  await Airship.show<boolean>(bridge => <ConfirmContinueModal bridge={bridge} title={title} body={body} />)
```

Comment:
This is incorrect per the task specs. Warning show show once per EVM wallet, not always. Need to move the config file to per wallet storage